engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/cache/SchemaCache.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.engine.avro.cache
+
+import java.io.Closeable
+
+import org.apache.avro.Schema
+
+import scala.concurrent.duration.Duration
+
+trait SchemaCache extends Closeable {
```

Comment:
I wonder if we need to specify that it is "schema" cache? Can it be just Simple cache with key as String and value as a generic parameter `T`? and move it to some util package?

Reply:
Yep, we can do it - it will be some base Cache for nk.